dashboard/config/locales/announcements.ar-SA.json

```diff
@@ -2,35 +2,44 @@
         },
-        "a3109564-513f-47c8-bdf3-d69a19f5149e": {
-          "buttonText": "اعرف المزيد"
+        "2b95b40d-3be9-445c-856a-879a6eceeea1": {
+          "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية."
         },
-        "9b45f4aa-6b0a-44d7-9813-8e724baca343": {
-          "buttonText": "اعرف المزيد"
+        "69e413fa-9081-4fac-8d73-9294b6fa1f11": {
+          "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية."
         },
-        "26f29937-fd4e-4738-8dfe-459f286636f4": {
-          "buttonText": "اعرف المزيد"
+        "1ff50ed3-0032-4ed0-b28c-376ce0f1c2ce": {
+          "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية."
         },
-        "b7bff9e1-22bd-4f95-bb95-68e91fd5b3a3": {
+        "633d1416-a7fb-44b6-a724-8479fd66f4fa": {
+          "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية."
+        },
+        "5972aa0b-1730-4977-82db-23cee2bda683": {
+          "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية."
+        },
+        "c89013b3-d0d5-4faa-842a-718092083fe6": {
+          "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية."
+        },
+        "79a249d7-fb90-4e9c-bd8b-e7884ad2034b": {
           "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية.",
           "buttonText": "تعلّم المزيد"
         },
-        "79a249d7-fb90-4e9c-bd8b-e7884ad2034b": {
+        "f1160bfc-3ae9-41a1-8ac9-1c9e7de4afeb": {
           "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية.",
           "buttonText": "تعلّم المزيد"
         },
         "b5b0c936-3bc6-4e69-8796-e205d103151f": {
           "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية.",
           "buttonText": "تعلّم المزيد"
         },
-        "f1160bfc-3ae9-41a1-8ac9-1c9e7de4afeb": {
+        "2e1b13ae-2ab8-4996-bf23-ca3e0fa1f8be": {
           "details": "بالإضافة إلى مناهجنا الدراسية، تقدم Code.org دورات تعلم احترافية ذاتية يقودها الميسرون للمعلمين الذين يرغبون في تطبيق CS Discoveries في فصولهم الدراسية.",
           "buttonText": "تعلّم المزيد"
```

Comment:
These don't seem to be sorted, so I'm guessing Crowdin reorders the given JSON? I hope they never upgrade their Ruby version.